Vagrantfile

```diff
@@ -56,5 +59,7 @@ setenforce 0
 systemctl stop firewalld
 sudo groupadd docker && sudo gpasswd -a ${USER} docker && sudo systemctl restart docker && newgrp docker
 usermod -aG docker vagrant
+sudo modprobe overlay
+echo "131072" | sudo tee /sys/module/nf_conntrack/parameters/hashsize
```

Comment:
Ah well seems alright... I would add that as a comment to the Vagrantfile it might be useful at one point to have that information somwhere.